FileProcessor/src/test/java/com/group7/FileProcessor/ScenarioTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/UnitTests/JUnit4TestClass.java to edit this template
+ */
+package com.group7.FileProcessor;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author keric
+ */
+public class ScenarioTest {
```

Comment:
Can you explain what this test class is supposed to do?

Reply:
It does nothing, I left it out of other commits, but automatically added it carelessly this time.